gcovr/writer/json.py

```diff
@@ -229,6 +229,8 @@ def _json_from_lines(lines):
 def _json_from_line(line):
     json_line = {}
     json_line["branches"] = _json_from_branches(line.branches)
+    if line.decisions:
+        json_line["decision"] = _json_from_decision(line.decisions)
```

Comment:
It just occurred to me that this field should be namespaced to avoid incompatibilities with the upstream gcov JSON format.

```suggestion
        json_line["gcovr/decision"] = _json_from_decision(line.decisions)
```

Would require corresponding changes to the JSON reader.